### PR TITLE
Fixing docstring for LCMV beamformer

### PR DESCRIPTION
**Pull request by britta-wstnr**
Clarifying the None option for both ``weight_norm`` and ``pick_ori``. 
Thanks @jasmainak and @agramfort for pointing this out!

### CODE REVIEW

File: mne/beamformer/_lcmv.py [hunk 1]
```diff
@@ -152,15 +152,18 @@ def make_lcmv(info, forward, data_cov, reg=0.05, noise_cov=None, label=None,
         If 'normal', rather than pooling the orientations by taking the norm,
         only the radial component is kept. If 'max-power', the source
         orientation that maximizes output source power is chosen.
+        If None, a vector beamformer is computed, combining the output for all
+        source orientations.
```
Comment (jasmainak) [inline mne/beamformer/_lcmv.py]: @britta-wstnr thanks for looking into this.

Are you sure this is correct? Take a look here: https://github.com/mne-tools/mne-python/blob/master/mne/beamformer/_lcmv.py#L396. This also seems to depend on what is in the forward.

Comment (britta-wstnr) [reply mne/beamformer/_lcmv.py]: You are right (maybe why it was left out before?). I will see how to rephrase it with taking the forward into account, thanks again!

**Comment by agramfort** [comment]
@britta-wstnr you have flake8 errors due to too long lines.

**Comment by britta-wstnr** [comment]
@agramfort fixed

**Comment by agramfort** [comment]
Thx

**Pull request merged**